_site/index.html

```diff
@@ -457,6 +458,59 @@ <h3>T-Z</h3>
         </section>
 
 
+	      <section class="block__word list">
+			    <p class="word__signal word__signal__avoid">
+				    avoid—Ableist slur
+			    </p>
+			    <p class="word__title" id="crazy">
+					  dumb
+				    <span class="word__speech">
+						  adjective
+				    </span>
+			    </p>
+    			<p class="word__definition">
+		      	unable or unwilling to speak; person with speech-related disabilities
+			    </p>
+			    <div class="block__type">
+				    <p class="word__type">
+					    Issues
+				    </p>
+				    <p class="word__breakdown">
+              Dumb has a specific meaning in the context of speech-related disabilities, but is often used as a derogatory adjective to describe a variety of other concepts.
+              It is also used as a synonym for low intelligence.
+            </p>
+			    </div>
+			    <div class="block__type">
+				    <p class="word__type">
+					    Impact
+				    </p>
+				    <p class="word__breakdown">
+              By using ableist language, we are perpetuating violence against people who experience mental or psychological disabilities.
+              Using this language perpetuates those systems and language of harm, regardless of our intent.
+            </p>
+			    </div>
+			    <div class="block__type">
+				    <p class="word__type">
+					    Usage Tip
+				    </p>
+				    <p class="word__breakdown">
+			        Be more specific.
+              Typically we can find an alternate definition by simply reflecting on what emotion we're really feeling.
+			      </p>
+			    </div>
+			    <div class="block__type">
+				    <p class="word__type">
+					    Alt Words
+				    </p>
+            <p class="word__breakdown">
+              <span class="word__alt">
+                dense, ignorant, uninformed, silly; nonspeaking, nonverbal
```

Comment:
I wonder here if should make this distinction more clear. I know you've separated the colloquial use of dumb and the medical use of dumb with a semi-colon.

I'm also wondering if we should encourage the use of dumb to connote nonspeaking and nonverbal? Had you done any research of this? (See below note).